README.md

```diff
@@ -220,12 +220,12 @@ Users can also change the configuration parameters of NanoMQ while booting. Howe
 
 In the `etc` directory of NanoMQ (`./nanomq/etc/nanomq.conf`), there is a 'nanomq.conf' configuration file. This file is different from 'config.cmake.in'. This 'nanomq.conf' allows you to configure broker when booting. Please be noted that if you start NanoMQ in the project's root directory, this file will be read automatically.
 
-
+Now NanoMQ features **MQTT bridge** by configuring `nanomq_bridge.conf` which in the same directory with `nanomq.conf`.
```

Comment:
Configure MQTT bridging in NanoMQ: by modifying `nanomq_bridge.conf`, which is in the same directory with `nanomq.conf`.